app/src/main/java/com/kamron/pogoiv/OcrHelper.java

```diff
@@ -509,12 +509,12 @@ private String getPokemonIdentifierFromImg(Bitmap pokemonImage) {
             return Optional.absent();
         }
 
-        Bitmap candyAmount = getImageCrop(pokemonImage, 0.65, 0.7, 0.15, 0.035);
+        Bitmap candyAmount = getImageCrop(pokemonImage, 0.60, 0.7, 0.20, 0.035);
         String hash = "candyAmount" + hashBitmap(candyAmount);
         String pokemonCandyStr = ocrCache.get(hash);
 
         if (pokemonCandyStr == null) {
-            candyAmount = replaceColors(candyAmount, true, 55, 66, 61, Color.WHITE, 200, true);
+            candyAmount = replaceColors(candyAmount, true, 55, 66, 61, Color.WHITE, 100, true);
```

Comment:
this should remove the candy completely
replaceColors(candyAmount, true, 68, 105, 108, Color.WHITE, 90, true);
![color replace](https://cloud.githubusercontent.com/assets/22281828/20452808/6c681ad6-ade0-11e6-9c5d-ad85930a1361.png)